pyro/distributions/torch.py

```diff
@@ -16,6 +17,9 @@
 
     try:
         _PyroDist = locals()[_name]
+        # Do not override existing __doc__
```

Comment:
This seems difficult to maintain. I think it would be simpler to
`if _name == 'Binomial': continue`.

Reply:
I am fine with that. However, if the class has an existing `__doc__`, it does not seem right to override that. Maybe we should append to it?

Reply:
Actually, lets get to it when we have more distributions that are overriding `__doc__`. Will change.

Reply:
The only distributions that should be in `locals()` should be distributions defined in this file, and those should be mere thin wrappers. As of your #961 there are no remaining wrappers, so this logic is unused. If we add wrappers in the future, I think we would not add a docstring. But I defer to your taste :smile: